### DISCUSSION TITLE
Is it illegal to be 🥴🍺 in a 🇬🇧 pub?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#765] [Galaxy Trivia] [answered]
**Bonus:** Justify

**Comment by peterbaccaro** [comment, marked as answer]
**It is not illegal to be drunk in a pub in the UK, but it is illegal to be disorderly or to cause a public disturbance while under the influence of alcohol**. 
The Licensing Act 1872 and the Licensing Act 2003 provide the legal framework for the operation of pubs and other licensed premises in the UK. These acts give licensees the right to refuse service to customers who are drunk, disorderly, or causing a public disturbance. It is also illegal to serve alcohol to someone who is already intoxicated. If someone becomes drunk and disorderly in a pub, they may be asked to leave by the staff, and the police may be called if the situation escalates.
